libtenzir/builtins/stores/feather.cpp

```diff
@@ -309,6 +309,54 @@ class plugin final : public virtual store_plugin {
       configuration{zstd_compression_level_});
   }
 
+  [[nodiscard]] auto serialize(inspector& f, const plugin_printer& x) const
+    -> bool override {
+    auto o = detail::overload{
+      [&x](std::reference_wrapper<caf::serializer> r) {
+        return r.get().object(x).fields();
+      },
+      [&x](std::reference_wrapper<caf::deserializer> r) {
+        return r.get().object(x).fields();
+      },
+      [&x](std::reference_wrapper<caf::binary_serializer> r) {
+        return r.get().object(x).fields();
+      },
+      [&x](std::reference_wrapper<caf::binary_deserializer> r) {
+        return r.get().object(x).fields();
+      },
+      [&x](std::reference_wrapper<caf::detail::stringification_inspector> r) {
+        return r.get().object(x).fields();
+      }};
+    auto obj = std::visit(o, f);
+    return f.apply(obj);
```

Comment:
`[&](auto f) { return f.get().object(x).fields(); }`

Reply:
Simplified the overloads.